tests/compile.sh

```diff
@@ -162,13 +131,12 @@ fi
 export BUILD_VERBOSE=1
 export BUILD_DIR
 export BUILD_JOBS
-export CCPP_SUITES
```

Comment:
If `CCPP_SUITES` is removed here it should also be removed from `build.sh`, and documentation (wiki) updated.

Reply:
@BinLiu-NOAA 
Please update `build.sh` and remove any and all references to `CCPP_SUITES` from `build.sh`.

Reply:
Done.